backstage.json

```diff
@@ -0,0 +1,3 @@
+{
+  "version": "0.4.4"
```

Comment:
Don't want to commit this file, can exclude this for now.

Reply:
Done , deleted